examples/quantizing_moe/README.md

```diff
@@ -61,7 +61,7 @@ oneshot(
     recipe=recipe,
     save_compressed=True,
     output_dir=output_dir,
-    overwrite_output_dir=True,
```

Comment:
what was this doing?

Reply:
Does nothing in oneshot unless in finetune flow because its a part of training_args

Reply:
@horheynm

Reply:
Following up on this, not sure why this conversation was resolved